### ISSUE TITLE
Database migration fails on Oracle with setting CLOB NULL (ORA-22859: invalid modification of columns)

### ISSUE THREAD

**Issue by kbower** [kbower/tickettest1#1530]
_Imported from trac ticket 1530.
Created by **kbower**
Opened in trac: Mar 15, 2012, 8:22:40 AM
Last modified in trac: Mar 16, 2012, 8:56:30 AM_
***
(filters table)
```
ALTER TABLE savedfilters RENAME COLUMN filter TO filter_031512del
2012-03-15 08:13:35,200 INFO sqlalchemy.engine.base.Engine {}
2012-03-15 08:13:35,338 INFO sqlalchemy.engine.base.Engine COMMIT
2012-03-15 08:13:35,384 INFO sqlalchemy.engine.base.Engine
ALTER TABLE savedfilters MODIFY (filter_031512del CLOB NULL)
2012-03-15 08:13:35,384 INFO sqlalchemy.engine.base.Engine {}
2012-03-15 08:13:35,497 INFO sqlalchemy.engine.base.Engine ROLLBACK
...
sqlalchemy.exc.DatabaseError: (DatabaseError) ORA-22859: invalid modification of columns
 '\nALTER TABLE savedfilters MODIFY (filter_031512del CLOB NULL)' {}
```

**Comment by kbower** [comment]
_**kbower** trac comment on Mar 16, 2012, 8:56:30 AM:_

(Changeset [[2443]](../commit/369183141791ee8631e5885a4fac3416eff7a25e))
 * Fix database migration to deal with most LOB changes (fixes #1530)

**Comment by kbower** [comment]
_**kbower** trac status change on Mar 16, 2012, 8:56:30 AM:_ **closed**
